pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -194,6 +198,17 @@ func (p *ProjectService) getKubernetesUpdateStrategy() *v1apps.RollingUpdateDepl
 		return &r
 	}
 
+	if cfg.Order == "start-first" {
+		if cfg.Parallelism != nil {
+			maxSurge := intstr.FromInt(cast.ToInt(*cfg.Parallelism))
+			r.MaxSurge = &maxSurge
+		}
+
+		maxUnavailable := intstr.FromInt(0)
```

Comment:
We could default that to K8s default of 25% here too?

Reply:
I think this might be a GitHub refresh issue, the change has already been made: https://github.com/appvia/kev/pull/519/commits/ed74c6cf7911465bb78f2e91f4526c4dffae9098#diff-178cbdee190dbd6e52b52aad3b5f30f05b300950c8d80fbb647df77a204ac55dR207